Fix FIX-ME request

TheTechRobo opened the pull request:
Also...
- Better README.md
- Remove some outdated comments
- Fix a few links

Notes:

- I used this source to help out: https://stackoverflow.com/a/4804039/9654083
- I used this source for making a "dummy" /etc/elive-version because I'm not at Elive right now: http://forum.elivecd.org/read.php?5,5255 (I googled `/etc/elive-version` to find this)
- Please enable the Issues section in the settings, so we can report bugs (it's disabled by default on forks)
- You say that you temporarily hardcoded it - I couldn't find this, so please, either explain to me or merge this PR and fix it yourself
- Finally, I'd suggest "squash and merge" (it's in the dropdown beside Merge pull request) because otherwise it may clutter up the commit history.

TheTechRobo committed:
Update README.md

TheTechRobo committed:
Potentially fix FIXME - will need to test

TheTechRobo committed:
Well THAT was not the issue

TheTechRobo committed:
Repair FIX-ME request.

TheTechRobo committed:
I probably won't, actually

Thanatermesis commented:
I see changes in the source code, it is verified that it correctly works with these changes especially into the related things that depends of this code ?

Commit:
Fix bug

Commit:
Fix bug

TheTechRobo commented:
@Thanatermesis for some reason it was working earlier and not now, thanks for catching that. currently workingon fixing the issue

> depends on this code

What depends on this code, so that I can test it?

TheTechRobo commented:
I found the issue in my fork by debugging. It's because there's a space between '383' and 'KB/s' and in my dummy file I didn't have that field. I'm not sure how to get around this - I'll keep thinking.

TheTechRobo commented:
Thinking - maybe I could use regex to look for `internet-average-speed: BLAH BLAH/s` and remove the space. Sounds complicated tho - I'll keep thinking.

TheTechRobo committed:
Untested - fix the issue in a "hacky" approach

Commit:
Merge down

Commit:
It seems to be fixed :D

Commit:
Remove debugging code

Commit:
Fix links:
Elive Forum
Replace wikipedia link with elive forum topic about it
Also...
Comment out useless stuff

TheTechRobo commented:
I fixed it using a "hacky" approach - it looks for a ValueError while doing it; if so it will combine the third value and the second value.

TheTechRobo commented:
Can you test this? It works for me, but I'd rather have two pairs of eyes rather than just one. :)
